recipes/ega-cryptor/ega-cryptor.py

```
@@ -0,0 +1,102 @@
+      (memory_options, prop_options, passthrough_options)
+    """
+    mem_opts = []
+    prop_opts = []
+    pass_args = []
+    exec_dir = None
+
+    for arg in argv:
+        if arg.startswith('-D'):
+            prop_opts.append(arg)
+        elif arg.startswith('-XX'):
+            prop_opts.append(arg)
```

Ruff

Combine `if` branches using logical `or` operator

Combine `if` branches

(SIM114)